btcstaking-tracker/btcslasher/slasher.go

```diff
@@ -132,10 +135,18 @@ func (bs *BTCSlasher) Start() error {
 		// start the subscriber to slashing events
 		// NOTE: at this point monitor has already started the Babylon querier routine
 		queryName := fmt.Sprintf("tm.event = 'Tx' AND message.action='%s'", messageActionName)
+		// subscribe to babylon fp slashing events
 		bs.finalitySigChan, startErr = bs.BBNQuerier.Subscribe(txSubscriberName, queryName)
 		if startErr != nil {
 			return
 		}
+		// subscribe to consumer fp slashing events
+		queryName = fmt.Sprintf("tm.event = 'Tx' AND message.action='%s'", consumerMessageActionName)
+		bs.equivocationEvidenceChan, startErr = bs.BBNQuerier.Subscribe(txSubscriberName, queryName)
+		if startErr != nil {
+			return
+		}
```

Comment:
Probably one can use a OR statement to subscribe to both in a single Subscribe routine

Reply:
tried doing it but it wasnt that straightforward as we then need to parse and sort the incoming messages. the current implementation seems a bit cleaner overall

Reply:
sounds good